### PR TITLE
Add an eyes test for the level details dialog

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Part 1 of https://codedotorg.atlassian.net/browse/PLAT-1092. I'm adding the eyes test and fixing the alignment issues separately as I've never written an eyes test before and want this to be a very targeted PR.

To add this test, I added a new lesson to allthemigratedthings and cloned some levels into it (that's a big reason this PR is so big). Let me know if there's a different way I should do this!)

The test adds 3 eyes checkpoints (the second will need an ignore region over part of the video):
Screen Shot 2021-06-16 at 8 08 17 AM

Screen Shot 2021-06-16 at 8 08 25 AM

Screen Shot 2021-06-16 at 8 08 33 AM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/ui/features/curriculum_platform/teacher_lesson_plan.feature [hunk 1]
```diff
@@ -83,3 +83,34 @@ Feature: Teacher Lesson Plan
...
+    And I wait until element ".modal-backdrop" is visible
+    And I see no difference for "standalone video preview"
+    And I click selector ".modal-body button:contains(Dismiss)"
+    And I wait until element ".modal-backdrop" is gone
+
+    And I click selector ".uitest-ProgressPill:nth(2)"
+    And I wait until element ".modal" is visible
+    And I wait until element ".modal-body button:contains(Dismiss)" is visible
+    And I wait until element ".modal-backdrop" is visible
+    And I see no difference for "standalone video preview"
```
Comment (davidsbailey) [inline dashboard/test/ui/features/curriculum_platform/teacher_lesson_plan.feature]: can you add a different name for this snapshot?

**Review by davidsbailey — APPROVED** [review]
nice work Bethany! Looks like good defensive UI/eyes test writing overall.

**Review by dmcavoy — APPROVED** [review]
Looks good after addressing Daves's comments. Nice work for a first eyes test!

File: dashboard/test/ui/features/curriculum_platform/teacher_lesson_plan.feature (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/test/ui/features/curriculum_platform/teacher_lesson_plan.feature]: oops yes -- good catch!

**Pull request merged**